addons/mail/static/tests/web/chatter_tests.js

```diff
@@ -315,7 +315,7 @@ QUnit.test("base rendering when chatter has no record", async () => {
     await contains(".o-mail-Message-body", { text: "Creating a new record..." });
     await contains("button", { count: 0, text: "Load More" });
     await contains(".o-mail-Message-actions");
-    await contains(".o-mail-Message-actions i", { count: 0 });
+    await contains(".o-mail-Message-actions i", { count: 1 });
```

Comment:
The translation action exist because the feature is enabled in the mock_server and the chatter message respect all the following conditions: https://github.com/odoo/odoo/pull/126555/files#diff-2666b2aa48ca5391cd3e6df5eb55dc517474d60964749e43f2d69ec46b94f121R320